shapely/_geometry.py

```diff
@@ -104,13 +104,16 @@ def get_dimensions(geometry, **kwargs):
 
     Examples
     --------
-    >>> get_dimensions(Geometry("POINT (0 0)"))
+    >>> from shapely import GeometryCollection, Point, Polygon
+    >>> point = Point(0, 0)
+    >>> get_dimensions(point)
     0
-    >>> get_dimensions(Geometry("POLYGON((0 0, 0 10, 10 10, 10 0, 0 0))"))
+    >>> polygon = Polygon(((0, 0), (0, 10), (10, 10), (10, 0), (0, 0)))
+    >>> get_dimensions(polygon)
     2
-    >>> get_dimensions(Geometry("GEOMETRYCOLLECTION (POINT(0 0), LINESTRING(0 0, 1 1))"))
-    1
-    >>> get_dimensions(Geometry("GEOMETRYCOLLECTION EMPTY"))
+    >>> get_dimensions(GeometryCollection([point, polygon]))
+    2
+    >>> get_dimensions(GeometryCollection())
```

Comment:
For this specific case, I find `"GEOMETRTCOLLECTION EMPTY` a bit more explicit / clear. 
Or maybe using `GeometryCollection([])` (with the empty list) also helps?